test/functional/rpcServer/commands/platform/getDataContractHandlerFactory.js

```diff
@@ -27,7 +27,7 @@ const Identity = require('@dashevo/dpp/lib/identity/Identity');
 const getDataContractFixture = require('../../../../../lib/test/fixtures/getDataContractFixture');
 const wait = require('../../../../../lib/utils/wait');
 
-describe('getIdentityHandlerFactory', function main() {
+describe('rpcServer.getIdentityHandlerFactory', function main() {
```

Comment:
Let's add nested describe instead of dot prefixing

Reply:
Done